lib/screens/lobby/list_of_created_games.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:niira/models/game.dart';
+import 'package:niira/screens/lobby/created_game_tile.dart';
+import 'package:niira/services/database/database_service.dart';
+import 'package:provider/provider.dart';
+
+class ListOfCreatedGames extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // build list of tiles that show a game's info
+    return StreamBuilder<List<Game>>(
+        stream: context.watch<DatabaseService>().streamOfCreatedGames,
+        builder: (context, snapshot) {
+          // show empty screen if stream hasnt come through yet
+          if (snapshot.data == null) {
+            return Container(
+              key: Key('list_of_created_games_empty_screen'),
```

Comment:
Nice!  I never thought to give the empty state a key so you can check for it... love it!